pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -0,0 +1,199 @@
+            <td>
+                <PrivilegedButton variant="link" buttonId="crypto-policy-button" tooltipId="tip-cpu-security"
+                                excuse={ _("The user $0 is not permitted to change crypto policies") }
+                                permission={ permission } onClick={() => setIsOpen(true)}>
+                    {currentCryptoPolicy}
+                </PrivilegedButton>
+                {isOpen &&
+                <CryptoPolicyDialog close={() => setIsOpen(false)}
+                          currentCryptoPolicy={currentCryptoPolicy}
+                          setCurrentCryptoPolicy={setCurrentCryptoPolicy}
+                />
+                }
+            </td>
+        </tr>
+    );
+};
+
+const CryptoPolicyDialog = ({
+    close,
+    currentCryptoPolicy,
+    setCurrentCryptoPolicy,
+}) => {
+    const [error, setError] = useState();
+    const [selected, setSelected] = useState();
+
+    const setPolicy = (reboot) => {
+        cockpit.spawn(["update-crypto-policies", "--set", selected], { superuser: "require", err: "message" }).then(() => {
+            setCurrentCryptoPolicy(selected);
+            if (reboot) {
+                cockpit.spawn(["shutdown", "--reboot", "now"], { superuser: "require", err: "message" })
+                        .catch(error => setError(error));
+            } else {
+                cockpit.spawn(["touch", stampfile], { superuser: "require", err: "message" }).then(() => {
+                    close();
+                });
+            }
+        })
+                .catch(error => {
+                    setError(error);
+                });
+    };
+
+    const policies = Object.keys(cryptopolicies).map(policy => {
+        return {
+            name: policy,
+            title: policy,
+            description: cryptopolicies[policy],
+            active: policy === currentCryptoPolicy,
+            recommended: policy === defaultpolicy,
+        };
+    });
+
+    const help = (
+        <Popover
+            id="crypto-policies-help"
+            bodyContent={
+                <div>
+                    {_("Crypto policies is a system component that configures the core cryptographic subsystems, covering the TLS, IPSec, SSH, DNSSec, and Kerberos protocols.")}
+                </div>
+            }
+            footerContent={
+                <Button component='a'
+                        rel="noopener noreferrer" target="_blank"
+                        variant='link'
+                        isInline
+                        icon={<ExternalLinkSquareAltIcon />} iconPosition="right"
+                        href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/security_hardening/using-the-system-wide-cryptographic-policies_security-hardening">
+                    {_("Learn more")}
+                </Button>
+            }
+        >
+            <Button variant="plain" aria-label={_("Help")}>
```

Comment:
I know the popover provides the functionality, and this should be focusable. It just looks weird to have a button without an action or link, is that legit?

Reply:
I think it looks alright, I'm not sure where else we would put the extra information.